lib/team.js

```diff
@@ -1,9 +1,12 @@
 // Generated by IcedCoffeeScript 108.0.11
```

Comment:
Weird that GH didn't detect this was a generated file. Any chance some unexpected line crept in?